src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1773,6 +1773,7 @@ To create/register a plugin, you have to :
 	#define USES_P211	// MPU6050
 	#define USES_P212	// MY9291
 	#define USES_P213	// VEML6070
+  #define USES_P240 // Vindriknting
```

Comment:
The Experimental build is no longer actively used, many plugins in this list have been migrated to regular plugins, but also renumbered to match the next available ID.
It could possibly go in one of the Collection_A..Collection_E builds (lines 1350..1425 in this file) (test with the ESP8266 builds to see if it fits) and/or the Climate build (around line 1640 in this file) (again test with the ESP8266 build to see if it fits).
Also, P144 should be enabled in the MAX build (add around line 1965 in this file)

Reply:
I added it to MAX. Still open to choose a set.

Reply:
I tried and it does fit in both Collection_E and Climate sets.

Reply:
Will update Collection_E and Climate. Please be sure to check my next pull request very well as I cannot test it on my Vindriktning itself. Somehow WiFi does not connect anymore since I rebased to mega.